src/components/Select/Select.stories.tsx

```diff
@@ -474,6 +475,57 @@ export const AdjustedWidth: StoryObj = {
   },
 };
 
+/**
+ * We lock the maximum height of the option list to 1/4 of the available screen height. Scrolling is allowed in the list, and
+ * keyboard navigation (showing the items off the edge of the screen) is handled when used.
+ */
+export const LongOptionList: StoryObj = {
+  args: {
+    ...Default.args,
+    defaultValue: 'test3',
+    className: 'w-60',
+    children: (
+      <>
+        <Select.Button>
+          {({ value, open, disabled }) => (
+            <Select.ButtonWrapper isOpen={open} shouldTruncate>
+              {value}
+            </Select.ButtonWrapper>
+          )}
+        </Select.Button>
+        <Select.Options>
+          {Array(30)
+            .fill('test')
+            .map((option, index) => (
+              // eslint-disable-next-line react/no-array-index-key
+              <Select.Option key={`${option}-${index}`} value={option + index}>
+                {option}
+                {index}
+              </Select.Option>
+            ))}
+        </Select.Options>
+      </>
+    ),
+  },
+  play: async (playOptions) => {
+    const canvas = within(playOptions.canvasElement);
+    const selectButton = await canvas.findByRole('button');
+
+    await openMenu(playOptions);
+    await userEvent.keyboard('{ArrowDown}{ArrowDown}{ArrowDown}{ArrowDown}');
+
+    await waitFor(async () => {
+      await expect(selectButton.getAttribute('aria-expanded')).toEqual('true');
+    });
```

Comment:
Do we need the wrapping `waitFor` because it takes time for the attribute to get set?

Reply:
I'm doing one final test on this. It might not be needed to wait actually; since the popover isn't in the normal DOM hierarchy, chromatic's snapshot is clamped to just the trigger/button. it was perfectly cropped out looks like.

The tell is that the chevron in the snapshot is pointing in the correct direction for it to be open 😅 . I can probably remove `@storybook/test` again, and remove this `waitFor`

Reply:
`waitFor` wasn't needed afterall, but i will keep `@storybook/test` since that let's us assert in the `play` functions (i think we can do more of that in places)